.github/workflows/cdn.yml

```diff
@@ -0,0 +1,36 @@
+name: CDN
+on:
+  push:
+    branches: master
+
+jobs:
+  release:
+    if: ${{startsWith(github.event.head_commit.message, 'Release')}}
```

Comment:
Shouldn't we use tags for this, assuming we tag our releases. Just a suggestion, I think you know better. :thinking:

Reply:
Release commits are always made by a script so this is quite reliable. Tags could be good but we do create like 20 of them each release, depending on which packages were updated

Reply:
Awesome :pray: